tests/test_authorize.py

```diff
@@ -59,7 +60,8 @@ def test_should_fail_if_not_in_list(self):
 class TestAuthorizeWithResource(TestAuthorize):
 
     def _create_fake_resource(self):
-        return FakeResource(set(('admin', 'principle', 'superintendent')))
+        authmap = {'on_get': ['admin', 'principle', 'superintendent']}
```

Comment:
A principle is a fundamental truth, a principal is something very important -- I think this should say the latter. (obvious nitpick)